src/processes/DimProcess.Library/DimProcessHandler.cs

```diff
@@ -243,13 +243,28 @@ public class DimProcessHandler(
             throw new UnexpectedConditionException("DownloadUrl must always be set");
         }
 
+        if (companyId == null)
+        {
+            throw new UnexpectedConditionException("CompanyId must not be null");
```

Comment:
All the other error messages are called "must always be set". Doesn't that fit here too?

Reply:
Done